{"name": "Получение заказов неавторизованным пользователем", "status": "broken", "statusDetails": {"message": "_pytest.fixtures.FixtureLookupError: ('ingredients', <SubRequest 'setup' for <Function test_get_orders_unauthorized>>)", "trace": "file /Users/artemkrivoshein/Diplom/Diplom_2/tests/test_get_user_orders.py, line 19\n      @allure.title('Получение заказов неавторизованным пользователем')\n      def test_get_orders_unauthorized(self, order_api):\nfile /Users/artemkrivoshein/Diplom/Diplom_2/tests/test_get_user_orders.py, line 7\n      @pytest.fixture(autouse=True)\n      def setup(self, order_api, ingredients, user_token):\nE       fixture 'ingredients' not found\n>       available fixtures: cache, capfd, capfdbinary, caplog, capsys, capsysbinary, class_mocker, cov, doctest_namespace, mocker, module_mocker, monkeypatch, no_cover, order_api, package_mocker, pytestconfig, record_property, record_testsuite_property, record_xml_attribute, recwarn, register_user, session_mocker, setup, tmp_path, tmp_path_factory, tmpdir, tmpdir_factory, user_api, user_data\n>       use 'pytest --fixtures [testpath]' for help on them.\n\n/Users/artemkrivoshein/Diplom/Diplom_2/tests/test_get_user_orders.py:7"}, "start": 1729435618469, "stop": 1729435618469, "uuid": "0267559f-8848-41e7-8b68-eebbf66df2fb", "historyId": "92bf968573314a866ca27467231d26bc", "testCaseId": "92bf968573314a866ca27467231d26bc", "fullName": "tests.test_get_user_orders.TestGetUserOrders#test_get_orders_unauthorized", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_get_user_orders"}, {"name": "subSuite", "value": "TestGetUserOrders"}, {"name": "host", "value": "MacBook-Air-Artem.local"}, {"name": "thread", "value": "50826-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_get_user_orders"}]}